docs/schema.md

```diff
@@ -324,6 +332,12 @@ platform information that doesn't apply equally to all listed versions and
 ranges, a separate entry with the same `package` in the `affected` array may be
 needed.
 
+The `severity` field is an optional element [defined here](#severity-field). 
```

Comment:
Sorry one more comment: Can you just move this text down to under the heading on line 411? That makes this text a bit more easily discoverable since it can be found from the ToC. 

The other text here around the relevant fields (e.g. "versions", "ranges") live here because they offer some context relevant to both of them.

Reply:
Makes sense @oliverchang ! All set